src/test/java/net/syscon/elite/service/support/StringWithAbbreviationsProcessorTest.java

```diff
@@ -0,0 +1,14 @@
+package net.syscon.elite.service.support;
+
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class StringWithAbbreviationsProcessorTest {
+    @Test
+    public void format() {
+        // Assert that all abbreviations are unchanged, even when there a multiple ones in a string
+        final var abbreviationsString = String.join(", ", StringWithAbbreviationsProcessor.ABBREVIATIONS);
+        assertThat(StringWithAbbreviationsProcessor.format(abbreviationsString)).isEqualTo(abbreviationsString);
```

Comment:
could have explicit tests - empty string, null, no match, multiple matches, match at end of string

Reply:
Because this tacks all the abbreviations in one place it tackles a few of the above, like end of string and multiple matches, but you're right that it will be better to make that explicit.